packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -19,6 +19,8 @@ export const template = html<TableRow>`
                 </${checkboxTag}>
             </span>
         `)}
+        <!-- This is needed to help align the cell widths exactly with the column headers. -->
```

Comment:
This comment ends up getting emitted in the shadow DOM on the page. Do we have a pattern for not doing that?

![image](https://user-images.githubusercontent.com/10500124/232905557-695c40f8-7063-43d4-8a54-8e48b8866555.png)

Reply:
Bleh. I'd vote to remove the comment. @mollykreis do you have another suggestion?

Reply:
Cam use the same pattern as we do for css (really for any js template string)
```js
${'' /* This is needed to help align the cell widths exactly with the column headers. */}
```